sockeye/inference.py

```diff
@@ -1568,6 +1551,7 @@ def _beam_search(self,
 
             # (7b) optionally save beam history
             if self.store_beam:
+                finished_or_inactive = mx.nd.clip(data=finished + inactive, a_min=0, a_max=1)
                 unnormalized_scores = mx.nd.where(finished_or_inactive,
                                                   scores_accumulated * self.length_penalty(lengths - 1),
```

Comment:
you also need to update this as otherwise we now had infinity for output-type beam_store